client/src/components/NavBar/NavBar.js

```diff
@@ -35,10 +35,12 @@ class NavBar extends Component {
         {this.props.showSearchBar ?
         <div className="NavBar-Search">
           <Select
+            key="search"
             closeMenuOnSelect={true}
             components={animatedComponents}
             value={this.props.selectedOptions}
             onChange={this.props.searchOnChange}
+            handleChange={(selectedValue,action) => this.props.searchOnChange(selectedValue, action)}
```

Comment:
Interesting 👍

Reply:
the key thing was in case it was a rendering. I probably could remove with no change.